packages/transport-tcp/TEST_RESULTS.md

```diff
@@ -0,0 +1,54 @@
+# TCP Connection Time Test Results
```

Comment:
There is no need to submit files like this.  Please add your findings to the PR and it'll be in the git history instead for interested parties.